### PR TITLE
User get notifications when dbt crawler run

### PR DESCRIPTION
**Pull request by elic-eon**
### 🤔 Why?

Some dbt models are imported using [project-dependencies](https://docs.getdbt.com/docs/collaborate/govern/project-dependencies). That means the dbt crawler may generate two entities with the same logical ID but we should not generate the imported one.

### 🤓 What?

- Filter models the `package_name` is not equal to `project_name` if `project_name` is set.

### 🧪 Tested?

Add test case.

### ☑️ Checks

- [x] My PR contains actual code changes, and I have updated the version number in `pyproject.toml`.

### CODE REVIEW

File: metaphor/dbt/artifact_parser.py [hunk 1]
```diff
@@ -427,6 +429,8 @@ def parse_manifest(
             .rsplit("/", 1)[-1]
             .split(".")[0]
         )
+        project_name = manifest_metadata.get("project_name", "")
```
Comment (mars-lan) [inline metaphor/dbt/artifact_parser.py:432]: If `project_name` is not set in metadata, all models will be filtered out?

Comment (elic-eon) [reply metaphor/dbt/artifact_parser.py:432]: no, if `project_name` is not set, all models will be extracted

Comment (elic-eon) [reply metaphor/dbt/artifact_parser.py:432]: [if project_name and node.package_name != project_name:](https://github.com/MetaphorData/connectors/pull/829/files#diff-a1df7e7750265ceca906dcd5962a36e71a550fa20dea40d2729a4ccde150bc4dR535)

File: metaphor/dbt/artifact_parser.py [hunk 2]
```diff
@@ -528,6 +532,8 @@ def parse_manifest(
                 node.unique_id,
                 node_name_getter(node),
             )
+            if project_name and node.package_name != project_name:
```
Comment (usefulalgorithm) [inline metaphor/dbt/artifact_parser.py:535]: I think we want to include referenced views, not the other way around?

File: metaphor/dbt/extractor.py [hunk 3]
```diff
@@ -67,11 +68,20 @@ async def extract(self) -> Collection[ENTITY_TYPES]:
...
-        entities.extend(self._virtual_views.values())
+        entities.extend(
+            map(
+                lambda item: item[1],
+                filter(
+                    lambda item: item[0] not in self._referenced_virtual_views,
+                    self._virtual_views.items(),
+                ),
+            )
+        )
```
[the remark this answers is not in the document]
Comment (elic-eon) [reply metaphor/dbt/extractor.py]: 👍

**Review by usefulalgorithm — APPROVED** [review]
lgtm

**Pull request merged**